src/mdx/versionedLinks.js

```diff
@@ -0,0 +1,22 @@
+const visitPromise = import("unist-util-visit");
+const expandVersionedUrl = require("./expandVersionedUrl");
+
+const resolve = async () => {
+  visit = (await visitPromise).visit;
+};
+
+resolve();
+
+const versionedLinks = (options) => {
+  const transformer = async (ast, vfile) => {
+    await visit(ast, "link", (node) => {
```

Comment:
This visits every single markdown link in the docs. I think I've provided enough early-exits in the `expandVersionedUrl` function that we don't have to worry about builds slowing down....that also might even be premature optimization on my part. But if we do notice build times go way up, there might be some micro-optimizations we can make here?

Reply:
We would see this on local builds too right? I may be the only person who builds locally anymore, but I tend to do that when I'm making significant link changes, particularly across multiple pages.

Reply:
Yes, if there's a performance degradation we'd see it on local builds too.

For what it's worth, and anecdotally, developing locally with `npm start` doesn't appear any slower than it used to.

Reply:
Big proponent of building locally before pushing anything @akeller  🙏